src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+    private String deadlineDateString;
+    private LocalDate deadlineDate;
+
+    Deadline(String deadlineDateString, String deadlineDetail) {
+        super(deadlineDetail);
+        try {
+            this.deadlineDate = Task.dateStringToDate(deadlineDateString);
+            this.deadlineDateString = deadlineDate.getMonth().toString().toLowerCase() + " " +
+                    deadlineDate.getDayOfMonth() + " " + deadlineDate.getYear();
+        }
+        catch (DateTimeException e) {
```

Comment:
Do note that the catch statement should start on the same line as the closing braces of try block.